com.jwestall.Forecast.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "com.jwestall.Forecast",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "cosmic-ext-forecast",
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--talk-name=com.system76.CosmicSettingsDaemon",
+      "--filesystem=xdg-config/cosmic:ro"
+    ],
+    "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin",
+      "build-args": ["--share=network"],
```

Comment:
can't use this. Build is sandbox and must be offline.

Reply:
Noted, will remove.